cai_causal_graph/__init__.py

```diff
@@ -30,7 +30,7 @@
     'VARIABLE_NAME',
 ]
 
-__version__ = '0.2.0'
+__version__ = '0.2.0.dev1'
```

Comment:
@matteo-causalens I commented on Slack but this was wrong! I need to delete the tag, remove the release artefacts from `pypi`, and make a new tag.

CC: @francesca-raimondi-causalens

Reply:
Fixed now. `0.2.0.dev1` tag has been deleted, artefacts removed from `pypi` and `0.2.1.dev0` has been released.